manifests/install.pp

```diff
@@ -187,7 +208,7 @@
     timeout => 1800,
   }
   # Install gitlab with the appropriate package manager (rpm or dpkg)
-  package { 'gitlab':
+  package { "$::gitlab::install::gitlab_pkg":
```

Comment:
You can't do that because previous versions called the package on the OS level gitlab.  Post 7.10.0 the package is now called gitlab-ce on the OS.  Soooooo, you have to have this variable to determine what it's called based upon the gitlab_branch in the manifest.